doc/changelog/02-specification-language/18834-master+use-extra-impargs-program-fixpoint-wf.rst

```diff
@@ -0,0 +1,6 @@
+- **Added:**
+  :cmd:`Program Fixpoint` with `measure` or `wf` now supports the
```

Comment:
Looks like the refman doesn't treat Program Fixpoint as its own command.
```suggestion
  :ref:`program_fixpoint` with `measure` or `wf` now supports the
```

Reply:
Thanks. I removed the `:cmd:` and added the `:ref:`.